tests/unit/algos/list/basic.js

```diff
@@ -65,4 +65,50 @@ describe('Basic listing algorithm', () => {
                                    Basic, { maxKeys: 1 }, logger);
         assert.deepStrictEqual(res, ['key1']);
     });
+
+    it('Shall support filtering custom attributes', () => {
+        const attr1 = {
```

Comment:
Since the tests embed a different set of parameters and are not really parameterized, you might as well set the attribute values close to the tested calls. Otherwise I would have suggested a `forEach()` loop to parameterized but I think it's not applicable here.

Reply:
not applicable